src/material-experimental/mdc-form-field/form-field.html

```diff
@@ -80,14 +80,15 @@
 
 <div class="mat-mdc-form-field-subscript-wrapper mat-mdc-form-field-bottom-align"
      [class.mat-mdc-form-field-subscript-dynamic-size]="subscriptSizing === 'dynamic'"
-     [ngSwitch]="_getDisplayedMessages()">
-  <div class="mat-mdc-form-field-error-wrapper" *ngSwitchCase="'error'"
+     *ngIf="_getDisplayedMessages() as displayedMessages">
+  <div class="mat-mdc-form-field-error-wrapper" *ngIf="displayedMessages === 'error'"
```

Comment:
I think that `_syncDescribedByIds` in `form-field.ts` will have to be updated too. Currently it's set up not to point to hints when they're hidden. It'll need a unit test too.

Reply:
There are several tests in the `input.spec.ts` that are checking the `aria-describedby` which is set by `_syncDescribedByIds`.

Reply:
Haven't forgotten, just busy! Thanks!